src/Taxjar/DependencyInjection/TaxjarOptions.cs

```diff
@@ -0,0 +1,25 @@
+#if NETSTANDARD2_0
+using System;
+
+namespace Taxjar.Options
+{
+    public class TaxjarOptions
+    {
+
+        /// <summary>
+        /// Taxjar token.
+        /// </summary>
+        public string ApiToken { get; set; }
+
+        /// <summary>
+        /// Is Sandbox?
+        /// </summary>
+        public bool IsSandBox { get; set; }
```

Comment:
In a future PR we are intending to change this to accept the URL directly. Since we are adding this, it would probably be better to go ahead and make this `ApiUrl` instead. Then the configuration for projects would be setting the URL instead of `IsSandbox`.

Reply:
What I have experienced from .net side of things it is easier to add a timestamp rather than figure out if it is seconds, min, hour... etc.

Reply:
From our testing of the APIs we discovered that Sandbox doesn't support:
- Address Validation
- Customer creation

Thus we ended up creating a factory with Named Options. I am okay with creating `ApiUrl` property that possibly will correspond to Named options.